metecho/api/models.py

```diff
@@ -258,6 +261,7 @@ class Repository(
     description = MarkdownField(blank=True, property_suffix="_markdown")
     is_managed = models.BooleanField(default=False)
     repo_id = models.IntegerField(null=True, blank=True, unique=True)
+    repo_image_url = models.URLField(blank=True)
```

Comment:
@wlonk Let's add a new Boolean field to allow skipping the repo image (either a `use_repo_image` that defaults `True`, or a `omit_repo_image` or something that defaults `False`).

Reply:
And now that I think about it, simplest might be to set `repo_image_url` to `None` (or `""`) in the serializer based on that new Boolean, so we don't need to expose it and add logic in the front end?